SofaKernel/modules/SofaLoader/src/SofaLoader/initSofaLoader.cpp

```diff
@@ -33,6 +34,10 @@ void initSofaLoader()
     static bool first = true;
     if (first)
     {
+        // msg_deprecated("SofaLoader") << "SofaLoader is deprecated; please load Sofa.Component.IO.Mesh instead.";
```

Comment:
```suggestion
        // msg_deprecated("SofaLoader") << "SofaLoader is deprecated. It will be removed at v23.06. Use Sofa.Component.IO.Mesh instead.";
```